integrationtests/gitcloner/clone_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/rancher/fleet/cmd/gitcloner/cmd"
 	"github.com/rancher/fleet/cmd/gitcloner/gogit"
 
+	docker_container "github.com/docker/docker/api/types/container"
```

Comment:
Aliases with underscores seem uncommon to me. We only do that for `_test`.

Couldn't find advice for import aliases, but for packages:

>  Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64. 
https://go.dev/doc/effective_go#package-names

Reply:
Thanks, underscores removed.